ios/FluentUI/Avatar/AvatarTokens.swift

```diff
@@ -13,17 +13,19 @@ open class AvatarTokens: ControlTokens {
     /// The size of the content of the `Avatar`.
     open var avatarSize: CGFloat {
         switch size {
-        case .xsmall:
+        case .size16:
             return 16
-        case .small:
+        case .size20:
+            return 20
+        case .size24:
             return 24
-        case .medium:
+        case .size32:
             return 32
-        case .large:
+        case .size40:
             return 40
-        case .xlarge:
```

Comment:
Do we know what impact this change is going to have for customers? It's different to adding just a new value. Will we need to add a size52 in the future? Do we know who's using this value now and how crucial it is to their layouts?

Reply:
Good point - I didn't dig into this. All I know is design requested this change to align our avatar sizes and naming with web avatars. Maybe @harrieshin has more info about this?